components/brave_shields/browser/brave_shields_util.cc

```diff
@@ -205,6 +205,20 @@ ControlType GetCookieControlType(Profile* profile, const GURL& url) {
       HostContentSettingsMapFactory::GetForProfile(profile), url);
 }
 
+void SetCookieWhitelist(HostContentSettingsMap* map, ControlType type,
```

Comment:
it's confusing to me that we have a method called `SetCookieWhitelist` that takes a ControlType. It seems to me that this is just an overloaded `SetCookieControlType` with a string pattern instead of a GURL, correct?

Reply:
I'd say `SetCookieControlTypeForPattern`? Overloads are often confusing and also not encouraged by CC.